Exercise_B/index.html

```diff
@@ -1,16 +1,20 @@
-<!DOCTYPE html>
-<html class="no-js">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-  <body>
-    <!-- CODE HERE -->
-
-    <script src="" async defer></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title></title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <!-- CODE HERE -->
+    <div class="blue Box"></div>
+    
+    <div class="green Box"></div>
+    <div class="red Box"></div>
+    
+    <script src="" async defer></script>
+  </body>
+</html>

```

Comment:
Hello Nicole, 
Great work with this, you can try to increase the size of the box as its quite small. But great work with this, and you have made good use of Float.